CONTRIBUTING.md

```diff
@@ -47,31 +47,21 @@ through the **project history**.  But also, we use the git commit messages to
 
 
 ### Commit Message Format
-Each commit message consists of a **header**, a **body** and a **footer**.  The header has a special
-format that includes a **type**, a **scope** and a **subject**:
+Each commit message consists of a **header** and a **body**.  The header has a special format that
+includes a **type**, a **scope** and a **subject**:
 
 ```
 <type>(<scope>): <subject>
 <BLANK LINE>
 <body>
-<BLANK LINE>
-<footer>
 ```
 
-The **header** is mandatory and the **scope** of the header is optional.
-
-Any line of the commit message cannot be longer 100 characters! This allows the message to be easier
-to read on GitHub as well as in various git tools.
-
-Note that in the future `gitcop` will be used to check if commits in pull
-request conform to commit message format, but since it can't be configured to
-have an optional `(<scope>)`, it will claim that messages without it are wrong,
-while they're perfectly fine.
+The **header** is mandatory . The **body** and the **scope** of the header are optional.
```

Comment:
You're right, I didn't read it that way, but the wording is extremely ambiguous.